Editor/Resources/CloudFormation/scenario2_single_fleet/lambda/results_request.py

```diff
@@ -60,3 +61,12 @@ def get_oldest_viable_game_session(gamelift, fleet_alias):
     )
     print(f"Received search game session response: {search_game_sessions_response}")
     return next(iter(search_game_sessions_response['GameSessions']), None)
+
+def create_player_session(gamelift, game_session_id, player_id):
+    print(f"Creating PlayerSession session on GameSession: {game_session_id}, PlayerId: {player_id}")
+    create_player_session_response = gamelift.create_player_session(
```

Comment:
Out of the scope: in the future we might want to queue and batch these calls (via create_player_sessions)